model/hard_constraint_model.py

```diff
@@ -0,0 +1,133 @@
+from gurobipy import *
+from xml_loader.shift_generation import *
+import sys
+def create_model():
```

Comment:
Is this necessary? To make sure everything is consistent we should aim to solve each problem only once.

Reply:
This is my own model. When merging I should create a new branch to remove files I use in testing. This is not relevant for the ALNS